### ISSUE TITLE
Question about scripts

### ISSUE THREAD

**Issue by VALKREE** [georgeto/g3dit#1]
Hello, in G3Edit, if we take weapons as an example, in gCItem_PS there are fields: "ArmorSet", "Spell" - how are they used and is there an example? Also in gCInteraction_PS there are fields: "Owner", "User", "AnchorPoint", "UsedByPlayer", "UseType", "ScriptUseFunc", "EnterROIScript", "ExitROIScript", "Spell" - it's also not clear how to apply this, and in particular: "ScriptUseFunc", "EnterROIScript", "ExitROIScript" are asked to enter bCString, and there is no list of possible strings anywhere.

**Comment by georgeto** [comment]
No, there is no dedicated example or tutorial for these probperties. My recommendation would be to use the property search of g3dit to find interesting entities/templates in the game's data, and use as an example / starting point. (see _Tools->Entity Search_ and _Tool->Template Search_)

Here I searched for all entities and templates that have a ScriptUseFunc property that is not empty:
![image](https://github.com/georgeto/g3dit/assets/9250103/f0a710e8-ee65-4bf6-a4e6-b27806eaf7db)

As another example lets look at gCInteraction_PS.UsedByPlayer. Turns out all entities and templates have UsedByPlayer set to false (i.e. the all have the same value).
So it it very likely that this is a property that is set at runtime. From its name and its placement in gCInteraction_PS  I would suspect that this is set to true when the player interactions with the entity, i.e. when the player has used the item.
So it could be used to track which entities were already used by the player, to be able to grant alchemy/intelligence bonus at a a bookstand only when reading it for the first time (UsedByPlayer was false before).
![image](https://github.com/georgeto/g3dit/assets/9250103/cf2bc756-751c-4ff0-b813-900d8033b36e)


For ScriptUseFunc there is some documentation: One usage of it is to set GameEvents when using interaction objects. See "1.7.2 Setzen von GameEvents beim Benutzen interaktiver Objekte" in the [CP_1_75_Handbuch_Ergaenzung_fuer_Modder_v4.0.pdf](https://www.worldofgothic.de/dl/download_478.htm).
